implemented public method getStatistic and private methods readFile, …

VladGural opened the pull request:
implemented public method getStatistic and private methods readFile, WriteFile

Resci commented on src/main/java/core/basesyntax/WorkWithFile.java:
```diff
@@ -1,8 +1,62 @@
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int SUPPLY = 0;
+    private static final int BUY = 1;
+    private static final int ACTION = 0;
+    private static final int QUANTITY = 1;
```
Add ending `_INDEX` for clarity

Resci commented on src/main/java/core/basesyntax/WorkWithFile.java:
```diff
@@ -1,8 +1,62 @@
+        String[] sourceStrings;
+        int[] results = new int[2];
+        File fileFrom = new File(fromFileName);
+        sourceStrings = readFile(fileFrom).split(" ");
```
What if we add a `comment` column and some cells contain `We need to buy more apples`?

Resci commented on src/main/java/core/basesyntax/WorkWithFile.java:
```diff
@@ -1,8 +1,62 @@
+        return resultString.toString().strip();
+    }
 
+    private void writeFile(File fileTo, String string) {
```
```suggestion
    private void writeFile(String fileName, String data) {
```

Resci commented on src/main/java/core/basesyntax/WorkWithFile.java:
```diff
@@ -1,8 +1,62 @@
+        writeFile(fileTo, resultString.toString());
+    }
+
+    private String readFile(File fileName) {
```
```suggestion
    private String readFile(String fileName) {
```

Resci commented on src/main/java/core/basesyntax/WorkWithFile.java:
```diff
@@ -1,8 +1,62 @@
+                currentString = reader.readLine();
+            }
+        } catch (FileNotFoundException e) {
+            throw new RuntimeException("Can't find file...", e);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read from file...", e);
```
```suggestion
            throw new RuntimeException("Can't find file", e);
        } catch (IOException e) {
            throw new RuntimeException("Can't read from file", e);
```

Resci commented on src/main/java/core/basesyntax/WorkWithFile.java:
```diff
@@ -1,8 +1,62 @@
+                results[BUY] += Integer.parseInt(splitString[QUANTITY]);
+            }
+        }
+        StringBuilder resultString = new StringBuilder();
```
Maybe better create a report in a separate method?

VladGural commented:
added prefix _INDEX to constant variables
checked create array int[] results
fixed method writeFile
fixed method readFile
fixed some comment string "Can't write to file..." ets.
reated new method report
